tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Features/Organisations Healthcheck.3538.feature

```diff
@@ -14,3 +14,15 @@ Scenario: 2. Smtp Server is down
 	Given The Smtp Server is down
 	When the dependency health-check endpoint is hit
 	Then the response will be Unhealthy
+
+@5648
+Scenario: 3. Database Server is up
```

Comment:
Please can you confirm what the test is doing here? It looks to me like we change the address of the health check endpoint indicating that the database is down. Would a better approach be to revoke the read and write privileges of the database then access the normal organisation health check URL?

Reply:
As per our discussion over the phone - 
The other address points to a 'carbon copy' of the API, with the only difference in them being access to the network - meaning the 'Broken API' can't talk to the db / smtp server.

Revoking r/w privileges wouldn't make the health check fail as the user would still be able to query 'SELECT 1;' and get a healthy response from the db. A way around that would be to drop the user before the test and then restore it afterwards, the same way we do in BAPI, however this approach tends to be more messy.
Given that the simplest way to 'break' the SMTP server is to use the broken API's endpoint, I thought doing the same for the database check would be appropriate as its the simplest solution that works.